CMakeLists.txt

```diff
@@ -197,6 +197,10 @@ if(APPLE)
     HPCC_ADD_SUBDIRECTORY(package)
 endif(APPLE)
 
+if(USE_SPARK)
```

Comment:
this might be standard cmake naming convention, but this flag could be more self documenting as INTEGRATE_SPARK ??

Reply:
USE_SPARK is the convention we use for including an optional feature in a package.  You can find examples of this in the commonSetup.cmake file, (ex. USE_PYTHON3).